src/Tools/AI Test Toolkit/src/AITCommandLineCard.Page.al

```diff
@@ -26,7 +26,7 @@ page 149042 "AIT CommandLine Card"
                 Caption = 'General';
                 field("AIT Suite Code"; AITCode)
                 {
-                    Caption = 'Test Suite Code', Locked = true;
+                    Caption = 'Test Suite Code';
                     ToolTip = 'Specifies the ID of the suite.';
```

Comment:
What about moving all page field tooltips to the table fields?

Reply:
There is no source table for this page.